mix.exs

```diff
@@ -40,33 +40,10 @@ defmodule Appsignal.Phoenix.MixProject do
     system_version = System.version()
     otp_version = System.otp_release()
```

Comment:
The `otp_version` variable is unused. (There's a warning for this directly on the GitHub UI... is it automatically parsing these warnings from the check output?) 

```suggestion
```